docs/Linking.md

```diff
@@ -1,6 +1,19 @@
 # Manual Linking
 
 ## iOS
+1. Open your project `.xcodeproj`
```

Comment:
Please split `.xcodeproj` linking and `pods`, as they're two separate methods to link it.

Reply:
Hi @Krizzu! :) Thanks for taking a look at this :)

I've changed the language a little bit and I think now the steps are more clear to follow.